mains/uci.cpp

```diff
@@ -157,15 +157,17 @@ int main() {
 				} else if (arg == "depth") {
 					input_stream >> depth;
 				} else if (arg == "movetime") {
-					input_stream >> time_ms;
+					int movetime;
+					input_stream >> movetime;
+					time_ms = movetime / 2;
```

Comment:
Fixes an issue where Rengar takes too long in the first move of a lichess bot game and the opponent aborts